bin/web/index.js

```diff
@@ -1,22 +1,72 @@
 const express = require('express')
+const os = require("os");
 const http = require('http')
 const path = require('path')
+const fs = require('fs')
 const Dalai = require("../../index")
 const app = express()
 const httpServer = http.Server(app);
 const start = (port, home) => {
-  const dalai = new Dalai(home)
-  dalai.http(httpServer)
-  app.use(express.static(path.resolve(__dirname, 'public')))
-  app.use(express.json());
-  app.use(express.urlencoded({ extended: true }));
-  app.set('view engine', 'ejs');
-  app.set('views', path.resolve(__dirname, "views"))
-  app.get("/", (req, res) => {
-    res.render("index")
-  })
-  httpServer.listen(port, () => {
-    console.log(`Server running on http://localhost:${port}/`)
-  })
+    const dalai = new Dalai(home)
+    dalai.http(httpServer)
+    app.use(express.static(path.resolve(__dirname, 'public')))
+    app.use(express.json());
+    app.use(express.urlencoded({ extended: true }));
+    app.set('view engine', 'ejs');
+    app.set('views', path.resolve(__dirname, "views"))
+    app.get("/", (req, res) => {
+        res.render("index")
+    })
+
+    const defaultDir = path.join(__dirname, "prompts");
```

Comment:
default prompts are saved in ./prompts (can be changed with a pull request)

user can create custom prompts in ~/user/dalai/config/prompts/ that will show up